pkg/frontend/adminactions/kubeactions.go

```diff
@@ -61,17 +65,35 @@ func NewKubeActions(log *logrus.Entry, env env.Interface, oc *api.OpenShiftClust
 		return nil, err
 	}
 
+	kubeInterface, err := kubernetes.NewForConfig(restConfig)
+	if err != nil {
+		return nil, err
+	}
+
 	return &kubeActions{
 		log: log,
 		oc:  oc,
 
 		gvrResolver: gvrResolver,
 
-		dyn:       dyn,
-		configcli: configcli,
+		dyn:           dyn,
+		configcli:     configcli,
+		kubeInterface: kubeInterface,
 	}, nil
 }
 
+func (k *kubeActions) KubeGetPodLogs(ctx context.Context, namespace string, name string, containerName string) ([]byte, error) {
```

Comment:
Same as in the interface above

```suggestion
func (k *kubeActions) KubeGetPodLogs(ctx context.Context, namespace, name, containerName string) ([]byte, error) {
```

Reply:
done